### PR TITLE
Add  legal links

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes # 176432447

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots

image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/core/src/LegalLinks/index.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const useStyles = makeStyles((theme) => ({
     },
     [theme.breakpoints.up("md")]: {
       "& > li": {
-        borderRight: `1px solid ${theme.palette.text.secondary}`,
+        // borderRight: `1px solid ${theme.palette.text.secondary}`,
```
Comment (gertie-sheshe) [inline packages/core/src/LegalLinks/index.js]: Commented code

**Review by gertie-sheshe — COMMENTED** [review]
Looks good 💯

File: packages/core/src/LegalLinks/index.js (hunk 1 above)
Comment (nerdlet) [reply packages/core/src/LegalLinks/index.js]: @gertie-sheshe So for this, I reverted it back to how it was just to stick with the convention we are using `Material ui` after yesterday's morning call.

**Review by KobbyMmo — APPROVED** [review]
looks good @nerdlet

**Pull request merged**